src/modules/mavlink/mavlink_main.cpp

```diff
@@ -749,17 +749,22 @@ Mavlink::get_free_tx_buf()
 	 */
 	int buf_free = 0;
 
+#if defined(CONFIG_NET) || defined(__PX4_POSIX)
+
 	// if we are using network sockets, return max length of one packet
 	if (get_protocol() == UDP || get_protocol() == TCP) {
-		return  1500;
+		return MAVLINK_MAX_PACKET_LEN;
```

Comment:
But shouldn't we fill up UDP frames?

Reply:
That's probably worth exploring for a large portion of what we do in mavlink, but currently each mavlink message is sent out (sendto) individually.

Reply:
I see.